html/signUp.html

```diff
@@ -0,0 +1,160 @@
+                    5 119 32 201 73 220 243 489 497 784 110 129 325 355 337 355 5 0 55 -41 111
+                    -91z m3100 -41 c381 -405 608 -732 699 -1008 27 -81 31 -110 32 -200 1 -95 -2
+                    -110 -26 -156 -117 -223 -520 -238 -1128 -42 -98 32 -373 133 -388 143 -1 2
+                    12 56 29 123 36 136 34 164 -13 188 -28 15 -34 15 -64 0 -25 -11 -36 -25 -43
+                    -53 -5 -21 -17 -67 -26 -103 -9 -36 -20 -71 -24 -78 -5 -10 -56 11 -198 82
+                    -105 52 -232 118 -282 146 l-92 52 168 106 c92 59 253 168 358 243 104 75 191
+                    135 193 134 1 -1 -7 -66 -17 -144 -23 -157 -19 -181 30 -201 34 -15 65 -6 84
+                    24 15 22 47 222 60 376 l7 75 131 107 c73 60 184 155 247 212 63 58 120 105
+                    127 105 6 1 67 -59 136 -131z m-2336 -565 c94 -68 248 -172 342 -232 l171
+                    -110 -91 -51 c-50 -28 -177 -94 -282 -146 -144 -72 -192 -92 -198 -82 -22 40
+                    -96 387 -124 583 -28 202 -28 188 -7 173 9 -7 95 -68 189 -135z m803 -509 c74
+                    -41 224 -119 332 -172 109 -53 198 -102 198 -109 0 -22 -110 -286 -152 -362
+                    -113 -207 -242 -353 -364 -412 -62 -30 -78 -33 -155 -34 -77 0 -91 3 -152 34
+                    -37 19 -92 57 -123 85 l-57 51 31 49 c81 127 60 289 -49 391 -63 59 -105 76
+                    -203 84 l-78 6 -19 50 c-10 28 -19 55 -19 61 0 5 64 40 143 77 78 37 228 114
+                    332 172 105 57 192 104 195 104 3 1 66 -33 140 -75z m-620 -566 c77 -53 98
+                    -130 58 -208 -25 -48 -89 -90 -138 -90 -36 0 -110 40 -134 72 -11 16 -21 49
+                    -24 80 -3 44 0 59 20 88 54 79 149 105 218 58z" />
+                            <path d="M2411 3115 c-192 -54 -348 -210 -405 -404 -20 -70 -21 -218 -1 -295
+                    53 -202 232 -375 432 -416 85 -18 244 -8 318 20 136 51 252 154 316 282 41 82
+                    59 159 59 258 0 99 -18 176 -59 258 -64 128 -179 230 -316 282 -78 30 -262 38
+                    -344 15z m330 -173 c84 -41 160 -117 202 -201 30 -62 32 -71 32 -181 0 -110
+                    -2 -119 -32 -181 -42 -84 -118 -160 -202 -202 -62 -30 -71 -32 -181 -32 -110
+                    0 -119 2 -181 32 -84 42 -160 118 -202 202 -30 62 -32 71 -32 181 0 110 2 119
+                    32 181 52 105 149 190 256 225 27 9 78 13 143 11 90 -2 107 -6 165 -35z" />
+                        </g>
+                    </svg>
+                </div>
+                <p class="left--content">
+                    Our customers are our gretest priority which hence our unique bussiness model.
+                </p>
+                <div class="footer-left--payment">
+                    <img src="/assests/image/rupay.png" alt="">
+                    <img src="/assests/image/visa.png" alt="">
+                    <img src="/assests/image/Paypal-logo.png" alt="">
+                    <img src="/assests/image/mastercard.png" alt="">
```

Comment:
Suggestion: Please provide alt description for accessibility checkpoint.

Reply:
ok will do